config/initializers/migrator/crosswalk_overrides.yml

```diff
@@ -49,5 +49,62 @@ overrides:
     function: return_nil
   - property: access_restrictions_attributes
     predicate: http://data.archiveshub.ac.uk/def/accessRestrictions
-    multiple: false
+    multiple: true
+    function: attributes_data
+  - property: is_volume
+    predicate: http://sw-portal.deri.org/ontologies/swportal#isVolume
+    multiple: true
+  - property: has_number
+    predicate: http://sw-portal.deri.org/ontologies/swportal#hasNumber
+    multiple: true
+  - property: biographical_information
+    predicate: http://www.rdaregistry.info/Elements/a/#biographicalInformation.en
+    multiple: true
+  - property: description_of_manifestation
+    predicate: http://www.rdaregistry.info/Elements/m/#descriptionOfManifestation.en
+    multiple: true
+  - property: form_of_work
```

Comment:
I think `attributes_data` properties have the `_attributes` suffix in the property name, so `form_of_work` would be `form_of_work_attributes`.